image/image.go

```diff
@@ -8,10 +8,24 @@ import (
 // Type must be a "Backing file format name string" that appears in QCOW2.
 type Type string
 
+type Extent struct {
```

Comment:
Needs godoc to clarify how this is different from clusters

Reply:
Right, will add.